roles/promtail/templates/promtail.yml.j2

```diff
@@ -0,0 +1,26 @@
+server:
+  http_listen_port: {{ promtail.ports.http_listen_port[type] }}
+  grpc_listen_port: {{ promtail.ports.grpc_listen_port[type] }}
+
+positions:
+  filename: {{ promtail_position_file_path }}
+
+client:
+  external_labels:
+    network: "perseverance"
+    host: "{{ ansible_default_ipv4.address }}"
+    vanity: "{{ promtail_node_vanity_name }}"
+    ss58Address: "{{ promtail_node_id }}"
+  url: https://loki-community.chainflip.xyz/loki/api/v1/push
+
+scrape_configs:
+  - job_name: systemd
+    static_configs:
+      - targets:
+          - localhost
+        labels:
+          __path__: /opt/{{ node_service_identifier }}/log/{{ node_service_identifier }}.log
```

Comment:
will need to add node suffix here after the changes

Reply:
Didn't get it

Reply:
after we add node type, directory structure will be:

chainflip-node
chainflip-engine

so the directory which just has "chainflip" needs to be changed to chainflip-node